daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -797,7 +797,19 @@ private[lf] object SBuiltin {
     }
   }
 
-  /** $tupd[field] :: Struct -> a -> Struct */
+  /** $tproj[field] :: Struct -> a */
+  final case class SBStructProjByName(field: Ast.FieldName) extends SBuiltinPure(1) {
```

Comment:
Why not adding a comment to explicate that this is a non-optimize form of `SBStructProj`

Reply:
Good idea. I'll do that.